filament/include/filament/IndirectLight.h

```diff
@@ -342,6 +343,68 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
     /** @deprecated use static versions instead */
     UTILS_DEPRECATED
     math::float4 getColorEstimate(math::float3 direction) const noexcept;
+
+    /**
+     * Sets IBL lookup options.
+     *
+     * @param options Options for IBL lookup settings.
+     */
+    void setIblOptions(IblOptions const& options) noexcept;
+
+    /**
+     * Gets IBL lookup options.
+     *
+     * @return IBL lookup options.
+     */
+    IblOptions const& getIblOptions() const noexcept;
+
+    /**
+     * Sets IBL type.
+     *
+     * @param iblTechnique Determine the IBL type (infinite, or finite box/sphere)
+     */
+    void setIblTechnique(const IblOptions::IblTechnique iblTechnique) noexcept;
```

Comment:
If the `IblTechnique` type is part of `IblOptions`, why have separate getters/setters here? Actually why aren't all the new APIs part of `IblOptions`? That would be consistent with other similar APIs.

Reply:
Using only `IblOptions` would allow to get rid of all the APIs you add below, making the PR much simpler.

Reply:
Fixed in b0f2ef543036f99a374befdcb44e38fbfa2b8df5 and 901f2a025aa0c1be79f2c56d51f7833d533bb4e4